1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
```

Comment:
Check this code

Sorted List not print

Reply:
_It's Working..._
It Prints....
![Screenshot_2021-06-21_19_49_29](https://user-images.githubusercontent.com/76242275/122777919-04ab5300-d29c-11eb-9f65-ff0907250183.png)